numba_cuda/numba/cuda/codegen.py

```diff
@@ -390,6 +390,10 @@ def add_linking_file(self, path_or_obj):
 
         self._linking_files.add(path_or_obj)
 
+    @property
+    def linking_files_as_obj(self):
```

Comment:
It seems a bit strange that the property isn't just called `linking_files`, but maybe I'm not understanding some nuance here.

Reply:
Right, I believed at one point I wanted to use this property to normalize object and paths into `LinkableCode`. But now this logic lives in `LinkableCode` factory functions.